### PR TITLE
Allow specifying the version of buildtools in bzlmod

### PR DESCRIPTION
**Pull request by hunshcn**
Resolve https://github.com/keith/buildifier-prebuilt/issues/86

### CODE REVIEW

**Review by cgrindel — COMMENTED** [review]
Could we add an example that exercises the `toolchains` bzlmod tag? We should then ensure that [the example runs with the integration tests](https://github.com/keith/buildifier-prebuilt/blob/008234b1d8c5ea5355b2cfa8810e95770739b62b/examples/BUILD#L34-L38).

**Comment by hunshcn** [comment]
I'm sorry that the test has failed before due to my unfamiliarity with the test and some accidents.

Now it should be fine.

**Comment by hunshcn** [comment]
All tests passed. Do you have time to have a look? @cgrindel @keith

**Review by cgrindel — APPROVED** [review]
LGTM. @keith Do you want to take a look before we merge?

File: examples/bzlmod_specify_assets/version_test.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+if "$1" --version | grep -q "4.2.5"; then
+  exit 0
+fi
+exit 1
```
Comment (brentleyjones) [inline examples/bzlmod_specify_assets/version_test.sh:6]: Nit: needs an extra newline.

**Comment by hunshcn** [comment]
ping @keith 

Although 7.3.1 has been updated, it may be meaningful in the future.

Or close this pr?

**Pull request merged**